### DISCUSSION TITLE
is 2GB the max

### DISCUSSION THREAD

**Discussion by layne-sadler** [git-lfs/git-lfs#4620] [Q&A]
the website says "a couple GB in size" is that the max?

**Comment by bk2204** [comment]
Git LFS does not have a practical maximum size for LFS objects (the limit is 2^63-1 bytes).  Obviously larger objects must be hashed, so having very large objects means that you will spend more time hashing them.

Your hosting provider may have a specific size limit.  For example, [GitHub has some size limits](https://docs.github.com/en/github/managing-large-files/versioning-large-files/about-git-large-file-storage).
